src/css/base.styl

```diff
@@ -12,25 +12,25 @@ body
     color: text-color
     background-color: black
     font-family: "Titillium Web", "Helvetica Neue", sans-serif
-    font-size: 13px
-    line-height: 18px
+    font-size: 1rem
+    line-height: 1rem
```

Comment:
Does this change mess up when there are icons on each line?

Reply:
Good catch on those.